base/libgit2/tree.jl

```diff
@@ -70,3 +70,9 @@ function Base.show(io::IO, te::GitTreeEntry)
     println(io, "Entry type: ", entrytype(te))
     println(io, "Entry OID: ", entryid(te))
 end
+
+function Base.show(io::IO, tree::GitTree)
+    println(io, "GitTree:")
+    println(io, "Owner: ", repository(tree))
```

Comment:
Is there a nice `show` method for `GitRepo` that we should take advantage of here?